lib/ansible/module_utils/basic.py

```diff
@@ -1255,7 +1280,7 @@ def _log_invocation(self):
             msg = 'Invoked'
 
         # 6655 - allow for accented characters
-        if isinstance(msg, unicode):
+        if not isinstance(msg, str):
             # We should never get here as msg should be type str, not unicode
             msg = msg.encode('utf-8')
```

Comment:
Do you want me to revert this change, to keep the test failing as a reminder?  Full syslog/journal fixing appears to be complicated enough to warrant splitting out into a separate pull request.